src/graphql/root/mutation/device-notification-token-create.ts

```diff
@@ -28,7 +28,7 @@ const DeviceNotificationTokenCreateMutation = GT.Field<
       const user = await User.findOne({ _id: toObjectId<UserId>(uid) })
       if (!user) throw Error("find user issue")
 
-      user.deviceToken.addToSet(deviceToken)
+      user.deviceToken.push(deviceToken)
```

Comment:
if you change this you need to validate that device token is not present in the array

Reply:
I see... perhaps there is another method `addToSet` is no longer present.